src/index.html

```diff
@@ -6,7 +6,43 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar--month-length--29 calendar--start-day--thu">
```

Comment:
```suggestion
      <div class="calendar calendar--month-length_29 calendar--start-day_thu">
```
Modifier syntax: block--modifier_value